Cargo.toml

```
@@ -27,7 +27,7 @@ iota-client = { git = "https://github.com/iotaledger/iota.rs", rev = "981136e911
 log = { version = "0.4.14", default-features = false }
 once_cell = { version = "1.8.0", default-features = false }
 rand = { version = "0.8.4", default-features = false }
-rocksdb = { git="https://github.com/iotaledger/rust-rocksdb", rev = "d2c579660e6c72ec1f14f0f2781cef5e3171d380", default-features = false, features = ["lz4"] }
+rocksdb = { version = "0.17.0", default-features = false, features = ["lz4"] }
```

I think we'll still need to use the Git repo since they haven't included https://github.com/rust-rocksdb/rust-rocksdb/commit/e2e8834975cc9f2cef63ae85d7ea376e7cdc816d in a release yet

```suggestion
rocksdb = { git = "https://github.com/rust-rocksdb/rust-rocksdb", rev = "f98e432623f339d55f7f3cb467dcca6dda62567a", default-features = false, features = ["lz4"] }
```